src/redux/localStorageUtils.ts

```diff
@@ -0,0 +1,88 @@
+export function createLocalStorageObject(name: string){
+    try {
+        let existing = localStorage.getItem(name);
+        if (!existing) {
+            setLocalStorageObject('notification', 'inProgressList', []);
+            setLocalStorageObject('notification', 'loadingList', []);
+            setLocalStorageObject('notification', 'erroredList', []);
+            setLocalStorageObject('notification', 'successList', []);
+            setLocalStorageObject('notification', 'isUpdate', "false");
+        }
+    }catch (error) {
+        console.log(error);
+    }
+};
+
+function setLocalStorageObject(name: string, key: string, value: any) {
+    let existing = localStorage.getItem(name);
+    let existingOne = existing ? JSON.parse(existing) : {};
+    existingOne[key] = value;
+    localStorage.setItem(name, JSON.stringify(existingOne));
+};
+
+export function addToLocalStorageObject(name: string, key: string, value: string) {
+    try {
+        let existing = localStorage.getItem(name);
+        let existingOne = existing ? JSON.parse(existing) : {};
+        if (existingOne[key]) {
+            if (existingOne[key].length >= 3)
+                existingOne[key].shift();
+            existingOne[key].push(value);
+        }
```

Comment:
@binduak can we extract L28-L30 to a different function

Reply:
@vinishad As the **if** check is specific to that method, I think it can be extracted if it can be used in multiple instances.